jiracli/cli.go

```diff
@@ -71,6 +71,9 @@ type GlobalOptions struct {
 	// location using the `pass` tool, if missing prompt the user and store in the PasswordDirectory
 	PasswordSource figtree.StringOption `yaml:"password-source,omitempty" json:"password-source,omitempty"`
 
+	// Cached password to avoid invoking password source on each API request
+	cachedPassword string
```

Comment:
While not a configuration option, this seemed like a good place to put this as `GetPass()` is a function defined with `GlobalOptions` as a receiver.

Open to move this somewhere else :)